drivers/lora/sx1276.c

```diff
@@ -550,6 +550,10 @@ static int sx1276_lora_init(struct device *dev)
 		return -EIO;
 	}
 
+	spi_cs.gpio_pin = GPIO_CS_PIN;
+	spi_cs.gpio_dt_flags = GPIO_CS_FLAGS;
+	spi_cs.delay = 0U;
```

Comment:
delay is initialized to 0 anyway, because variable is declared as static. Is there a reason to specify that explicitly?

Reply:
Yeah, I realised that as well. Most other drivers that I looked at seem to do it explicitly though. Happy to drop it if you think it is a problem.

Reply:
It is just a preference, not a blocker :)